debian/control

```diff
@@ -73,8 +73,9 @@ Depends: binutils,
          ${misc:Depends},
          ${shlibs:Depends}
 Recommends: btrfs-tools, ceph-mds, librados2, libradosstriper1, librbd1
-Replaces: ceph-common (<< 0.78-500), python-ceph (<< 0.92-1223)
-Breaks: python-ceph (<< 0.92-1223)
+Replaces: ceph-common (<< 0.78-500), python-ceph (<< 0.92-1223),
+	  ceph-test (<< 0.94-1322)
+Breaks: python-ceph (<< 0.92-1223), ceph-test (<< 0.94-1322)
```

Comment:
These version numbers need to be altered to reflect "git describe" on the current tip of the hammer branch. So instead of "0.94-1322", it should be "0.94.1-something"